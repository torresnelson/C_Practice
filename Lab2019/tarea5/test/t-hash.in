# prueba de tiempo hash
tiempo_hash 1000000
Fin
