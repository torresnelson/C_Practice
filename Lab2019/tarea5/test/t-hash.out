1>#  prueba de tiempo hash.
2>prueba actualizar. 
prueba existe_asociacion, valor_en_hash. 
3>Fin.
